package.json

```diff
@@ -35,6 +35,7 @@
     "dedent": "^0.7.0",
     "execa": "^0.8.0",
     "find-parent-dir": "^0.3.0",
+    "fs-promise": "^2.0.3",
```

Comment:
[`fs-promise` is deprecated](https://github.com/kevinbeaty/fs-promise). Moreover, we could just use `pify` to promisify `fs`:

```js
λ node
> const pify = require('pify')
undefined
> const fs = pify(require('fs'))
undefined
> fs.readFile('./.lintstagedrc', 'utf8').then(console.log)
> {
    "linters": {
        "*.js": [
            "eslint --fix",
            "git add"
        ],
        ".*{rc, json}": "jsonlint"
    }
}
```